src/api/app/views/shared/_buildresult_box.html.erb

```diff
@@ -48,8 +48,8 @@
         url: '<%= url_for(:controller => controller, :action => 'rpmlint_result') %>',
         data: {
           <%= raw("'project': '#{project}',") if defined?(project) %>
-	  <%= raw("'package': '#{package}',") if defined?(package) %>
-	  <%= raw("'index': '#{index}',") if defined?(index) %>
+          <%= raw("'package': '#{package}',") if defined?(package) %>
```

Comment:
That wouldn't make a difference in terms of security. My concern was that a user might update the name of a package to some malicious javascript i.e. an XSS attack but it seems that the package name is validated to only allow letters/numbers in so I dont think that will be possible:

https://github.com/adrianschroeter/open-build-service/blob/SSL/src/api/app/models/package.rb#L1003-L1015

That said maybe it would still be a good idea to change the varaible from `package` to `package_name` so its clear to others what exactly is stored in that variable?